pageimport/README.md

```diff
@@ -1,24 +1,122 @@
-# pageimport
+# Page Creation
 
-This component builds a portlet with maven to create a set of pages under a given unique name of a page and adds the WCM portlet to it.
+This project contains a [Java utility](https://github.com/HCL-TECH-SOFTWARE/dx-portlet-development-utilities/blob/main/pageimport/src/main/java/com/hcl/dx/sample/util/Utility.java) to create a single page using a default or by selecting an existing page template. The project gives an example source code that shows how to use the Controller and Model SPI calls related to WCM, for which the API calls will be handled inside of a JSP JSCR286 portlet that provides the graphical user interface.  
 
-# Building the application
+It requires the parent page unique name under which the new page needs to be created. You may find the unique name under the Manage Pages portlets, e.g. shown here for parentPage.
+You need to provide the page name and friendly URL for your new page, and then there are two samples, explained here.
 
-Follow the commands in the below order to generate the Portlet war file.
+## Utility Features
 
-- Run ```mvn package``` to generate the pageimport DX Portlet war file.
-This will generate war file for you to install. You can find the war file under the target directory.
+### Create new page with Web Content Viewer configured to a content sample
 
-# Deploying the application
+If you provide the Content UUID, it will create a page based on the default page template which has a Web Content Viewer portlet deployed. It will set this portlet with your content you provided.
```

Comment:
```suggestion
If you provide the Content UUID, the portlet will create a page based on the default page template and add a Web Content Viewer portlet. It will configure the content viewer portlet with the content you provided.
```